test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -0,0 +1,45 @@
+package seedu.addressbook.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList.DuplicatePersonException;
+
+public class TestUtil {
+    /**
+     * Creates an address book containing the given persons.
+     * @throws DuplicatePersonException if two or more given persons are the same
+     */
+    public static AddressBook createAddressBook(Person... persons) throws DuplicatePersonException {
+        AddressBook addressBook = new AddressBook();
+        
+        for (Person person : persons) {
+            addressBook.addPerson(person);
+        }
+        
+        return addressBook;
+    }
+    
+    /**
+     * Creates a list of persons.
+     */
+    public static List<ReadOnlyPerson> createList(Person...persons) {
+        List<ReadOnlyPerson> list = new ArrayList<ReadOnlyPerson>();
+        
+        for (Person person : persons) {
+            list.add(person);
+        }
+        
+        return list;
+    }
+    
+    /**
+     * Creates an exact copy of the original address book.
```

Comment:
May want to mention how deep the copy is. e.g. is it going to contain same Person objects or copies of those Person objects?